src/cloud/docker/docker-containers.md

```diff
@@ -0,0 +1,140 @@
+---
+group: cloud-guide
+title: Docker container architecture
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+The [Magento Cloud Docker repository][docker-repo] contains build information to create a Docker environment with the required specifications for Magento Cloud. You can customize the Docker containers available in the repository and add more as needed.
+
+Magento Cloud Docker generates the `docker-compose.yml` file to the required specifications. Then, you use docker-compose to create the the container instances, build, and deploy the {{site.data.var.ee}} site.
+
+## Service versions
+
+Magento Cloud Docker references the `.magento.app.yaml` and `.magento/services.yaml` configuration files to determine the services you need. When you start the Docker configuration generator using the `ece-docker build:compose` command, you can override a default service version with the following optional parameters:
+
+| Name       | Service   | Key  | Available Versions | Notes
+| ------------- | ---------- | ---------- | ------------------ |------------------
+| [build] | Build Container |   |   | PHP Container, runs build process
```

Comment:
This problem is caused by the rearchitecture and the PR submission from @aepod . Need to do some further rearchitecting. For example, I think the information about the service version options might belong on the Docker Quick Reference page.  I think the Container overview, CLI and Service topic need to be reorganized to minimize repetition and to provide a cleaner overview with links to instructions.  All the override info can be handled in one topic--perhaps updating the Docker container architecture to add a section about customizing using .magento.app.yaml and using the override file with examples, so we don't have to repeat that information in each container section.

I think the options for specifying service versions information might fit better on the Docker Quick Reference page. 


Interim fix:
- Add a CLI containers section to the Docker container architecture topic, and moved the CLI containers to a table in that section. The node container does not quite fit because it has a service version.